<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OlympiadViewer</class>
 <widget class="QWidget" name="OlympiadViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Результаты олимпиады</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="filterLayout">
     <item>
      <widget class="QLabel" name="label_school">
       <property name="text">
        <string>Школа:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="schoolComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_class">
       <property name="text">
        <string>Класс:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="classComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="resultTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Логин</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ФИО</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Баллы</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>